### PR TITLE
BNP Paribas: Add explanation on the further progress

### PR DESCRIPTION
**Pull request by rugk**
Also, they use a Nextcloud instance for providing that data, but I guess that's not relevant here…

### CODE REVIEW

File: companies/consorsfinanz.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     ],
     "suggested-transport-medium": "email",
     "comments": [
-        "Erbittet datenschutzspezifische Anfragen per E-Mail zu stellen."
+        "Erbittet datenschutzspezifische Anfragen per E-Mail zu stellen. Als Antwort erhält man einen Link und muss die Hotline anrufen, sich authentifiizieren und ein Einmalpassword zum Zugriff auf den Link erfragen."
```
Comment (rugk) [inline companies/consorsfinanz.json:35]:
```suggestion
        "Erbittet datenschutzspezifische Anfragen per E-Mail zu stellen. Als Antwort erhält man einen Link und muss die Hotline anrufen, sich authentifizieren und ein Einmalpassword zum Zugriff auf den Link erfragen."
```

**Comment by baltpeter** [comment]
Thanks for the note!

Would you mind instead submitting that as a comment on the [company page](https://www.datenanfragen.de/company/consorsfinanz)? That's where we want to encourage users to share their experience with the privacy practices of the companies.

In fact, the existing comment in the record that you added to shouldn't be there, either. This is just one of our oldest records back from when we hadn't yet established the `suggested-transport-medium` key. :D

**Comment by rugk** [comment]
Done, removed the comment now here.

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**